### DISCUSSION TITLE
Fluid style tag not working

### DISCUSSION THREAD

**Discussion by lucabertoni** [OrchardCMS/OrchardCore#14216] [Q&A]
Good morning,

I am using the style tag to load css but it is not being rendered:

```html<!DOCTYPE html>
<html lang="{{ Culture.Name }}">
    <head>
        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />

        {% style src:"~/GB_Theme/vendor/bootstrap-5.0.2-dist/css/bootstrap.min.css" %}

        {% script src:"~/GB_Theme/vendor/bootstrap-5.0.2-dist/js/bootstrap.bundle.min.js" %}

        <title>{{ "PageTitle" | shape_new | shape_stringify }}</title>
    </head>
```

While the script tag is rendered:

```html
<head>
        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">

        <script src="/GB_Theme/vendor/bootstrap-5.0.2-dist/js/bootstrap.bundle.min.js?v=fh8VA992XMpeCZiRuU4xii75UIG6KvHrbUF8yIS_2_4"></script>

        <title>asdf</title>
    </head>
```

Version: OrchardCore 1.6.0

Thank you.

**Comment by hishamco** [comment]
Could you please try

```html
 {% style name:"bootstrap", version:"5.1.3" %}
```

**Comment by lucabertoni** [comment]
Good morning @hishamco,

>  {% style name:"bootstrap", version:"5.1.3" %}

the same, not rendered.

**Reply by hishamco** [reply]
Strange!! We already use it in our templates. I need to investigate

**Comment by ns8482e** [comment]
To me it doesn't make sense to use style tag in liquid with src, you could just use link tag with href

The proper way to use style tag is to define your resources using ResourceManiest with a name and provide a name in your liquid style tag as @hishamco suggests.
